lib/std/process.zig

```diff
@@ -431,6 +431,20 @@ pub fn hasEnvVarConstant(comptime key: []const u8) bool {
     }
 }
 
+/// On Windows, `key` must be valid UTF-8.
+pub fn hasNonEmptyEnvVarConstant(comptime key: []const u8) bool {
```

Comment:
While you're here, do you feel like renaming the `*Constant` functions in this file to `*Comptime` for consistency with the rest of std?

Reply:
Okay, I believe I did in c37e3058a848501e6014d42fc412980434713d19

Reply:
Hmm, the more I think about it, the more unsure I am about this suggestion. I don't like the term "constant" because of confusion with `const`, but "comptime" is also fairly confusing in this particular context.

On second thought, let's maybe leave this alone until we can think of a better naming pattern for situations like this. Sorry for the churn.

Reply:
No worries, I just reverted it and squashed to a single commit, force pushed below in 44da56819b1029fd237b270f7ed5daaa73069db1